### PR TITLE
Surgeries that are done without anesthetic or painkillers now have a chance to cause extra damage

### PR DESCRIPTION
**Pull request by SomeguyManperson**
## About The Pull Request

Surgeries now have a chance (default 50%) to trigger a flinch effect, which causes extra damage and bleeding
This can be mitigated by medium levels of drunkenness, or neutralized entirely with a painkiller (nitrous, morphine, or miner's salve)
Being asleep in general reduces the chance of triggering a flinch effect by 20%, and failing the step increases it by 50%
This does not affect mechanical limbs since they are likely built significantly better for interior access

## Why It's Good For The Game

Good????? lol

## Changelog

:cl:
add: surgeries to organic limbs that are done without anesthetic or painkillers now have a chance to cause extra damage
balance: medical cyborgs now have access to morphine in their hypospray
/:cl:

### CODE REVIEW

**Comment by Leon9621** [comment]
Finally, anesthetic will be real in 2024

**Comment by Anticept** [comment]
"fuckup_damage" "commit_malpractice" "ouchie_mod" I love these proc and var names

Fucking up should also trigger screaming in the patient and mood debuff.

**Comment by tantspants** [comment]
I don't want to sound like a /tg/ player but I don't this improves anything.

File: code/modules/surgery/surgery_step.dm [hunk 1]
```diff
@@ -225,10 +240,29 @@
...
+	ouchie_mod *= clamp(1-target.drunkenness/SURGERY_DRUNK_MOD, 0, 1) //Drunkenness up to 50% (points? idk) will improve chances of avoiding horrible pain and suffering
+	if(target.stat == UNCONSCIOUS) //Being "normally" asleep (or getting choked out) will SLIGHTLY improve your chances since it's intuitive behavior barring access to anything else
+		ouchie_mod *= 0.8
+	if(!success) //Failing the surgery increases the chance of causing harm significantly
+		ouchie_mod *= 1.5
+	var/final_ouchie_chance = SURGERY_FUCKUP_CHANCE * ouchie_mod
+	if(!prob(final_ouchie_chance))
+		return
+	. = TRUE
+	user.visible_message("<span class='boldwarning'>[target] flinches, bumping [user]'s [tool ? tool.name : "hand"] into something important!</span>", "<span class='boldwarning'>[target]  flinches, bumping your [tool ? tool.name : "hand"] into something important!</span>")
```
Comment (thgvr) [inline code/modules/surgery/surgery_step.dm]: Span macros please

**Comment by Martinpachu** [comment]
does this affect IPCs?

**Comment by SomeguyManperson** [comment]
> does this affect IPCs?

No, mechanical limbs are unaffected

**Comment by Martinpachu** [comment]
> > does this affect IPCs?
> 
> No, mechanical limbs are unaffected

maybe there could be something interesting done like setting an IPC to a low power setting or something to prevent the whole tend-wounds-in-the-fields issue they have but that's outside the scope of this PR

**Review by rye-rice — APPROVED** [review]

**Comment by dessysalta** [comment]
> I don't want to sound like a /tg/ player but I don't this improves anything.

Not the PR author here, but I'll still add my two cents based on what I think the message is:

People should have an incentive to hit up medical ships for anything more than some bruises—it's why they exist. I would argue that not every ship should be self sufficient and there's a reason 90% of them aren't outfitted with giant high-tech medbays with defibs and stasis beds. This better incentivizes interaction with aid ships (who may otherwise have nothing to do, which eats their time and resources ICly and OOCly) as well as provides some realism in the form of "why in the hell are you performing Literal Actual Brain Surgery on a table with a pocket crowbar".

Or, at least, non-medical ships should look into some strong booze/painkillers if they're going to be opening people up.

**Comment by Anticept** [comment]
A critical downside to relying on medships is that they would have to be there and available in the first place. I do like the idea of ship specialization... but unless its consistent... it might as well not exist. Players don't like to rely on unreliable sources.

One thought that was brought up is someday have an outpost medical bay, but it wouldnt be cheap.

**Comment by Leon9621** [comment]
> A critical downside to relying on medships is that they would have to be there and available in the first place. I do like the idea of ship specialization... but unless its consistent... it might as well not exist. Players don't like to rely on unreliable sources.
> 
> One thought that was brought up is someday have an outpost medical bay, but it wouldnt be cheap.

FWIW, anesthetics are (currently) available for purchase for 500 credits. Push comes to shove, you can get guaranteed surgery stuff for cheap cheap.

**Comment by Anticept** [comment]
> > A critical downside to relying on medships is that they would have to be there and available in the first place. I do like the idea of ship specialization... but unless its consistent... it might as well not exist. Players don't like to rely on unreliable sources.
> > One thought that was brought up is someday have an outpost medical bay, but it wouldnt be cheap.
> 
> FWIW, anesthetics are (currently) available for purchase for 500 credits. Push comes to shove, you can get guaranteed surgery stuff for cheap cheap.

and honestly I think this is fair. right now, medships are charging exorbitant fees sometimes for services. Sometimes, you're not going to get much from a rescue and cleaning them out for whatever they have left is messed up. There needs to be an offset.

**Comment by SomeguyManperson** [comment]
The purpose here is to give people good pretense to use some form of painkillers for any intensive surgery, which is why alternative solutions like alcohol and miner salve are reliable

File: code/modules/surgery/surgery_step.dm [hunk 2]
```diff
@@ -116,8 +124,15 @@
 			if(failure(user, target, target_zone, tool, surgery, fail_prob))
 				play_failure_sound(user, target, target_zone, tool, surgery)
 				advance = TRUE
-			if(chem_check_result)
-				return .(user, target, target_zone, tool, surgery, try_to_fail) //automatically re-attempt if failed for reason other than lack of required chemical
+		if(!HAS_TRAIT(target, TRAIT_SURGERY_PREPARED) && target.stat != DEAD && !IS_IN_STASIS(target) && fuckup_damage) //not under the effects of anaesthetics or a strong painkiller
+			if(!HAS_TRAIT(user, TRAIT_SURGEON)) // The patient needs more mouse bites
```
Comment (FalloutFalcon) [inline code/modules/surgery/surgery_step.dm:128]: abductors are currently the only thing that gets this, it also lets them do ANY surgery i think.

**Pull request closed without merging**